examples/stock/company.v.py

```diff
@@ -1,21 +1,23 @@
 # Own shares of a company!
 company: address
-avail_shares: num
-total_shares: num
-price: wei_value
+total_shares: currency_value
+price: num (wei / currency)
 
```

Comment:
Are those getters `@constant` e.g. cost zero gas? because the data is available to be read?